x-pack/legacy/plugins/monitoring/server/kibana_monitoring/lib/send_bulk_payload.js

```diff
@@ -3,13 +3,58 @@
  * or more contributor license agreements. Licensed under the Elastic License;
  * you may not use this file except in compliance with the Elastic License.
  */
+import moment from 'moment';
+import { chunk, get } from 'lodash';
+import {
+  MONITORING_SYSTEM_API_VERSION,
+  KIBANA_SYSTEM_ID,
+  KIBANA_STATS_TYPE_MONITORING,
+  KIBANA_SETTINGS_TYPE,
+} from '../../../common/constants';
 
-import { MONITORING_SYSTEM_API_VERSION, KIBANA_SYSTEM_ID } from '../../../common/constants';
+const SUPPORTED_TYPES = [KIBANA_STATS_TYPE_MONITORING, KIBANA_SETTINGS_TYPE];
+export function formatForNormalBulkEndpoint(payload, productionClusterUuid) {
+  const dateSuffix = moment().format('YYYY.MM.DD');
```

Comment:
Should we maybe use UTC `moment.utc().format('YYYY.MM.DD')` here? Just in case their server is not UTC

Reply:
Yes, very good point! Thanks!